doc/content/api/reference/grpc/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: "gRPC"
+description: "Reference list of gRPC API"
+weight: 2
+---
+
+A full list of gRPC API reference per service is listed in this guide.
```

Comment:
Can you make sure that subsection tiles are either not shown at all or that **more** tag is being used? It looks like this now
![image](https://github.com/TheThingsIndustries/lorawan-stack-docs/assets/65215579/7e283f8f-b86a-483d-8265-6bf64ab5a7d3)
![image](https://github.com/TheThingsIndustries/lorawan-stack-docs/assets/65215579/be731dae-e6af-4f50-a556-f54f54970d4c)

Reply:
This was always the case but I can indeed fix it.